packages/neon/neon_notifications/lib/src/widgets/notification.dart

```diff
@@ -22,45 +22,53 @@ class NotificationsNotification extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ListTile(
-      title: Text(notification.subject),
-      subtitle: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          if (notification.message.isNotEmpty)
-            Text(
-              notification.message,
-              overflow: TextOverflow.ellipsis,
-            ),
-          RelativeTime(
-            date: tz.TZDateTime.parse(tz.UTC, notification.datetime),
-          ),
-          if (notification.actions.isNotEmpty)
-            Row(
-              children: notification.actions
-                  .map(
-                    (action) => NotificationsAction(
-                      action: action,
-                    ),
-                  )
-                  .toList(),
+    return Dismissible(
+      key: Key(notification.notificationId.toString()),
+      direction: DismissDirection.startToEnd,
+      background: Container(color: Colors.red),
+      onDismissed: (direction) {
+        onDelete();
+      },
+      child: ListTile(
+        title: Text(notification.subject),
+        subtitle: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            if (notification.message.isNotEmpty)
+              Text(
+                notification.message,
+                overflow: TextOverflow.ellipsis,
+              ),
+            RelativeTime(
+              date: tz.TZDateTime.parse(tz.UTC, notification.datetime),
             ),
-        ]
-            .intersperse(
-              const SizedBox(
-                height: 5,
+            if (notification.actions.isNotEmpty)
+              Row(
+                children: notification.actions
+                    .map(
+                      (action) => NotificationsAction(
+                        action: action,
+                      ),
+                    )
+                    .toList(),
               ),
-            )
-            .toList(),
-      ),
-      leading: NeonUriImage(
-        account: NeonProvider.of<Account>(context),
-        uri: Uri.parse(notification.icon!),
-        size: const Size.square(largeIconSize),
-        svgColorFilter: ColorFilter.mode(Theme.of(context).colorScheme.primary, BlendMode.srcIn),
+          ]
+              .intersperse(
+                const SizedBox(
+                  height: 5,
+                ),
+              )
+              .toList(),
+        ),
+        leading: NeonUriImage(
+          account: NeonProvider.of<Account>(context),
+          uri: Uri.parse(notification.icon!),
+          size: const Size.square(largeIconSize),
+          svgColorFilter: ColorFilter.mode(Theme.of(context).colorScheme.primary, BlendMode.srcIn),
+        ),
+        onTap: notification.link.isNotEmpty ? () => context.go(notification.link) : null,
+        onLongPress: onDelete,
```

Comment:
So now we have two ways to delete the notification? That's a bit strange UX wise and